packages/utilities/psammead-locales/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "@bbc/psammead-locales",
   "version": "2.1.0",
   "description": "A collection of locale configs, used in BBC World Service sites",
+  "main": "dist/index.js",
```

Comment:
when psammead is built these files dont exist :( Im not sure what the IE11 is, maybe the `"sideEffects": false,` fixes it? Not sure why though :(  Sorry I cant be more helpful

Reply:
Happy to try merging just the `"sideEffects": false,` and see if that helps?

Reply:
yeah adding just this `"sideEffects": false` should fix it.
Testing it now, to be sure

Reply:
Does anyone know if there is a reason why sideEffects: false should not be specified for all packages?

Reply:
@dr3 adding just the `sideEffects: false` flag does the trick,
Tested it out locally

Reply:
@jamesdonoh We only really added it where needed, alot of packages are accessed via a nested `package.json` so doesnt need it at the base `package.json`, we just missed it with moment